ProcessLib/Parameter/Parameter.h

```diff
@@ -54,6 +54,24 @@ struct Parameter : public ParameterBase
     //! Returns the parameter value at the given time and position.
     virtual std::vector<T> const& getTuple(
         double const t, SpatialPosition const& pos) const = 0;
+
+    //! Scalar value parameter access. The number of components of this
+    //! parameter must be 1.
+    double operator()(double const t, SpatialPosition const& x) const
```

Comment:
As I said, I am against `operator()`. This looks nice, but completely hides your intentions in code using this method.
Please rather use ordinary methods.